docs-chef-io/content/inspec/resources/lxc.md

```diff
@@ -11,32 +11,32 @@ platform = "linux"
     parent = "inspec/resources/os"
 +++
 
-Use the `lxc` Chef InSpec audit resource to test information about Linux containers. LXC is a command-line client for LXD that manage your LXD instances (containers and VMs). The tests are against the container's information obtained on `lxc info [container-name]`. `lxc` resource allows testing if the container exists or is in running status.
-
+Use the `lxc` Chef InSpec audit resource to test the information about Linux containers. LXC is a command-line client for LXD that manages your LXD instances (containers and virtual machines). The tests are against the container's information obtained on `lxc info [container-name]`. `lxc` resource allows the testing if the container exists or is in running status.
 
 ## Availability
 
 ### Installation
 
-This resource is distributed along with Chef InSpec itself. You can use it automatically.
+This resource is distributed with Chef InSpec.
 
 ## Syntax
 
-A `lxc` Chef InSpec audit resource allows testing if the container exists or is in running status.
+An   `lxc` Chef InSpec audit resource allows testing if the container exists or is in running status.
 
     describe lxc("linux-container-name") do
       it { should exist }
       it { should be_running }
     end
+
 where
 
-- `exist` and `be_running` are matchers of this resource
+- `exist` and `be_running` are matchers of this resource.
```

Comment:
I think we should remove this line. As in the matchers we covered this.

Reply:
@dkumaras please look into this.

Reply:
@ahasunos - Shall I remove this line? When I look into 'matchers.md' file, there is no mention of these variables.

Reply:
I do not see any harm in the where clause explaining that `exist` and `be_running` are the matchers in the example. It could be kept as it is in my opinion. 
Although, if it feels redundant it can be removed.